policyengine/countries/uk/default_reform.py

```diff
@@ -87,14 +87,64 @@ def create_default_reform() -> ReformType:
         for name, variable in baseline_system.variables.items()
     }
 
+    class property_wealth(Variable):
```

Comment:
will all these variables eventually be part of openfisca-uk directly?

Reply:
Yes, though I think it's fine to include in policyengine until then?